src/app/pages/TopicPage/HierarchicalGrid/index.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { string, shape, arrayOf, number, oneOfType, oneOf } from 'prop-types';
+import {
+  GEL_GROUP_1_SCREEN_WIDTH_MAX,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_2_SCREEN_WIDTH_MAX,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '#psammead/gel-foundations/src/breakpoints';
+import Promo, { MEDIA_TYPES } from '#components/Promo';
+import { DESKTOP, TABLET, MOBILE, SMALL } from './dataStructures';
+
+const PromoList = styled.ul`
+  padding: 1.5rem;
+  margin: 0;
+  display: grid;
+  grid-gap: 10px;
+  grid-template-columns: repeat(4, 1fr);
```

Comment:
It looks like 'desktop first styling' here, so the desktop breakpoint is default. Could these media queries be switched round so the smallest breakpoint is the default as per: https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/Styles.md#responsive-styles-prefer-a-mobile-first-approach-to-styling

Reply:
Yup. I'll be looking to switch to JoRo's improved method of implementing media queries